torch/_torch_docs.py

```diff
@@ -595,10 +595,13 @@ def merge_dicts(*dicts):
            r"""
 atan2(input, other, out=None) -> Tensor
 
-Returns a new tensor with the arctangent of the elements of :attr:`input`
-and :attr:`other`.
+Returns a new tensor with the arctangent of the elements of ``input``
+divided by ``other``.
 
-The shapes of :attr:`input` and :attr:`other` must be
+.. math::
+    \text{{out}}_{i} = \tan^{-1}(\text{{input}}_{i} / \text{{other}}_{i})
```

Comment:
this seems underspecified?  Doesn't atan2 specify unique values?  (see the numpy documentation).

Reply:
Fixed using a somewhat more concise way...